From 3cbd2fc69160f5c21b66445279fcb31c22e29915 Mon Sep 17 00:00:00 2001
From: John Kacur <jkacur@redhat.com>
Date: Fri, 27 Oct 2023 15:22:30 -0400
Subject: [PATCH 3/3] rt-tests: Remove remaining unnecessary texts after adding
 SPDX licenses

Remove remaining unnecessary texts after adding SPDX licenses

Signed-off-by: John Kacur <jkacur@redhat.com>
---
 src/signaltest/signaltest.c   |  4 ----
 src/sigwaittest/sigwaittest.c | 14 --------------
 2 files changed, 18 deletions(-)

diff --git a/src/signaltest/signaltest.c b/src/signaltest/signaltest.c
index 4737c253b1af..5412c50f7e17 100644
--- a/src/signaltest/signaltest.c
+++ b/src/signaltest/signaltest.c
@@ -5,10 +5,6 @@
  *
  * (C) 2007 Thomas Gleixner <tglx@linutronix.de>
  *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License Version
- * 2 as published by the Free Software Foundation;
- *
  */
 
 #include <fcntl.h>
diff --git a/src/sigwaittest/sigwaittest.c b/src/sigwaittest/sigwaittest.c
index 818e3a8e680a..55855769c63b 100644
--- a/src/sigwaittest/sigwaittest.c
+++ b/src/sigwaittest/sigwaittest.c
@@ -5,20 +5,6 @@
  *
  * Copyright (C) 2009 Carsten Emde <C.Emde@osadl.org>
  *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation either version 2 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307,
- * USA.
  */
 
 #include <stdio.h>
-- 
2.42.0

